File: Basic Coding Problems/02 If - Elses/032 Write a program to input angles of a triangle and check whether triangle is valid or not/C/triangle_validity.c
#include <stdio.h>

int main() {
    int angle_a, angle_b, angle_c;

    printf("Enter the angles of the triangle: ");
    scanf("%d %d %d", &angle_a, &angle_b, &angle_c);

    if (angle_a + angle_b + angle_c == 180 && angle_a > 0 && angle_b > 0 && angle_c > 0) {
        printf("The triangle is valid.");
    } else {
        printf("The triangle is not valid.");
    }

    return 0;
}

File: Basic Coding Problems/02 If - Elses/032 Write a program to input angles of a triangle and check whether triangle is valid or not/Java/triangle_validity.java
import java.util.Scanner;

public class triangle_validity {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the angles of the triangle: ");
        int angleA = scanner.nextInt();
        int angleB = scanner.nextInt();
        int angleC = scanner.nextInt();

        if (angleA + angleB + angleC == 180 && angleA > 0 && angleB > 0 && angleC > 0) {
            System.out.println("The triangle is valid.");
        } else {
            System.out.println("The triangle is not valid.");
        }
    }
}